README.rst

```diff
@@ -395,6 +395,32 @@ is a polite ping/enquiry.
       proposal with the implementation status (i.e. ticket URL and the
       first version of GHC implementing it.)
 
+      Concretely, a committee member must perform the following steps:
+
+      1. Add a new commit on top of the PR branch that:
+
+         a. Changes the filename of the proposal to correspond to the PR number.
+
+         b. Updates any metadata fields that may have changed in ``master`` since
+            the PR branch split off.
+
+         c. Fills in these metadata fields as appropriate, including changing "is discussed"
+            to "was discussed".
+
+      2. Merge the PR branch into master, and push.
+
+      3. Update the PR description:
+
+         a. Update the "Rendered" link to point to the permanent home
+         of the accepted proposal.
+
+         b. Add the text "The proposal has been accepted; the following discussion is mostly of historic interest."
```

Comment:
Actually, I leave the description alone and add

```
The [proposal](https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/… has been accepted; the following discussion is mostly of historic interest.

---
```
above. Nobody has to look below the `--`. More uniform.

Reply:
Also, remove `(under review)` from the PR title if it is there